<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1821</width>
    <height>964</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>1041</width>
      <height>431</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1401</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>168</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>8</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Следующий день</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>6</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Предыдущий день</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>6</pointsize>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Високосный?</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>8</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Номер недели</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>7</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Кол-во дней до</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>6</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>871</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="Add">
     <attribute name="title">
      <string>Добавить дату</string>
     </attribute>
     <widget class="QPushButton" name="addDate">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить дату</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="DateAddText">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2024</year>
        <month>3</month>
        <day>23</day>
       </datetime>
      </property>
      <property name="time">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>16</hour>
        <minute>38</minute>
        <second>56</second>
        <year>1752</year>
        <month>9</month>
        <day>14</day>
       </datetime>
      </property>
      <property name="minimumDate">
       <date>
        <year>1752</year>
        <month>9</month>
        <day>14</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2024</year>
        <month>3</month>
        <day>23</day>
       </date>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>35</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите дату для добавления</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="chn">
     <attribute name="title">
      <string>Изменить дату</string>
     </attribute>
     <widget class="QPushButton" name="changeDate">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>240</y>
        <width>311</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Изменить дату</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>491</width>
        <height>51</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string> Введите номер для изменения</string>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="cursorPosition">
       <number>28</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDateEdit" name="DateChangeText">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>85</y>
        <width>451</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите дату для изменения</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="birth">
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <attribute name="title">
      <string>Дней до Дня Рождения</string>
     </attribute>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>421</width>
        <height>81</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QDateEdit" name="DateBirthdayText">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::DaySection</enum>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>721</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>До вашего дня рождения осталось столько дней:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите вашу дату Рождения</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="openfile">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>40</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savefile">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>130</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>openfile</zorder>
   <zorder>savefile</zorder>
   <zorder>table</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1821</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
